packages/shift-next-routes/src/express/account-handler.js

```diff
@@ -249,9 +202,22 @@ function extractCustomerId (req, data) {
   }
 }
 
+function handleErrorResponse(error,req, res) {
+  const response = error.response
+  switch (response.status) {
+    case 404:
+    case 422:
+      req.log && req.log.debug({ status: response.status, errors: response.data.errors })
+      return res.status(response.status).send(response.data.errors)
+    default:
+      return res.status(response.status).send(response.data)
+  }
+}
+
 async function assignCartToUser (req, res) {
   if (!req.session.customerId) return
   const cartId = req.signedCookies.cart
+  req.log && req.log.info({ msg: 'assigning cart to user', cartId, customerId: req.session.customerId })
```

Comment:
totally personal opinion but i believe this should be debug

Reply:
Same reason as above really - audit trail.  but happy to downgrade this one if anyone has strong feelings